### DISCUSSION TITLE
How to display lectures of 1960 mass e.g 08-01r (its only displaying 08-01 which is 1955)

### DISCUSSION THREAD

**Discussion by d4h0y0sr** [DivinumOfficium/divinum-officium#3983] [General]
Hi everyone, i have issues because i see in the files the file 08-01r that has the correct texts of the mass for 1960 rubrics, but when i try in the website to display it, it only shows the 1955 lectures. I dont know if i have to select something else in the UI or if there is an issue why i can not see the correct information because according to the missal 1960 i have the lectures are: lectio !Heb 11:33-39 and evangelium !Luc 12:1-8

![imagen](https://github.com/user-attachments/assets/a91291dc-1923-45c1-b021-3220db9adb5f)

**Comment by APMarcello3** [comment]
The website, for 1960 4th class feasts (also known as commemorations), defaults to the Mass of the preceding Sunday, which is always an option.

**Reply by d4h0y0sr** [reply]
So is there any way i can visualize the 4th class feast? usually the priests celebrate it regularly so its not very useful for me to use the preceding sunday.

**Reply by APMarcello3** [reply]
You would have to change the setting to Reduced 1955 or Divino Afflatu, generally speaking.

**Reply by d4h0y0sr** [reply]
That is not a valid solution in this case, the lectures of 1955 and DA are not the same of the 1960 for the 1st of August :/
